### PR TITLE
Indicate how the SDK determines App Store installs.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/424020071714917

Also includes presentation/flow + screenshot updates for the SDK Feature Settings coverage, and enhancements for the intermediate topic that now includes a Dex count graphic.

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://55-bb-docs.surge.sh

File: sdk/README.adoc [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
 - link:usage_tracking.adoc[Usage Tracking]
 - link:feature_settings.adoc[Feature Settings]
 - link:integration.adoc[Manual Installation (iOS)]
 - link:api.adoc[SDK API]
+
+The SDK includes a lot of functionality in a small package; including
+the SDK adds less than 1 MB to your app's size.
+
+The following chart visualizes the number of methods available in the
+SDK for Android (the iOS version is very similar):
```
Comment (coryalder) [inline sdk/README.adoc]: Don't think this is relevant, or true, for iOS. Android has the dex limit thing, so they care about the number of methods, but iOS doesn't have any analogous limit. Also the SDK has +700 methods (rough estimate).

Would suggest just skipping the "(the iOS version is very similar)" bit. :)

Comment (bbewan) [reply sdk/README.adoc]: Ah, thanks. I'll fix that up.

Comment (bbewan) [reply sdk/README.adoc]: https://github.com/buddybuild-public/docs.buddybuild.com/pull/57

**Pull request merged**